app/services/process_feed_service.rb

```diff
@@ -163,7 +163,7 @@ def mentions_from_xml(parent, xml)
 
         url = Addressable::URI.parse(link['href'])
 
-        mentioned_account = if TagManager.instance.local_domain?(url.host)
+        mentioned_account = if TagManager.instance.web_domain?(url.host)
```

Comment:
No no no, this is the opposite of what you want. Web domain = mastodon.example.com. You want your users to be on example.com. Revert this one